src/System.Private.ServiceModel/src/System/ServiceModel/Security/SecurityUtils.cs

```diff
@@ -791,15 +791,15 @@ static internal bool IsDefault(NetworkCredential credential)
 
             internal static string GetCurrentUserIdAsString(NetworkCredential credential)
             {
-#if FEATURE_NETNATIVE
-                // There's no way to retrieve the current logged in user Id in UWP apps
-                // so returning a username which is very unlikely to be a real username;
-                return "_______****currentUser****_______";
-#else
+#if SUPPORTS_WINDOWSIDENTITY
                 using (WindowsIdentity self = WindowsIdentity.GetCurrent())
                 {
                     return self.User.Value;
                 }
+#else
+                // There's no way to retrieve the current logged in user Id in UWP apps
+                // so returning a username which is very unlikely to be a real username;
+                return "_______****currentUser****_______";
```

Comment:
How about a name like "____CURRENTUSER_NOT_AVAILABLE____" or something more informative?  Also the comment uses UWP.